bigchaindb/common/transaction.py

```diff
@@ -661,8 +661,8 @@ def create(cls, creators, recipients, metadata=None, asset=None):
                     - Multiple inputs and outputs.
 
             Args:
-                creators (:obj:`list` of :obj:`str`): A list of keys that
-                    represent the creators of the asset created by this
+                tx_signers (:obj:`list` of :obj:`str`): A list of keys that
+                    represent the signers of the asset created by this
```

Comment:
The `tx_signers` are the signers of this CREATE transaction, not the "signers of the asset" :smile: